src/omnisharp/utils.ts

```diff
@@ -81,7 +81,7 @@ export async function requestWorkspaceInformation(server: OmniSharpServer) {
         if (!blazorDetectionEnabled && response.MsBuild.Projects.some(project => isBlazorWebAssemblyProject(project))) {
             // There's a Blazor Web Assembly project but VSCode isn't configured to debug the WASM code, show a notification
             // to help the user configure their VSCode appropriately.
-            vscode.window.showInformationMessage('Your VSCode is not configured to debug Blazor web assembly applications.', 'Learn more', 'Close')
+            vscode.window.showInformationMessage('Additional setup is required to debug Blazor WebAssembly apps.', 'Learn more', 'Close')
```

Comment:
Not a big deal at all, but if you're open to adding further tweaks, I'd suggest:

```suggestion
            vscode.window.showInformationMessage('Additional setup is required to debug Blazor WebAssembly applications.', 'Learn more', 'Close')
```

Reply:
Oh I now see Dan specifically recommended the `apps` phrasing. Seems a bit unusual to me.

I would find `applications` more normal in this context but don't want to block this PR on such a minor detail. @danroth27 was this intentional?

Reply:
I like applications better as well. WIll commit the suggestion